scripts/jenkins/ardana/ansible/roles/bootstrap_ardana/tasks/main.yml

```diff
@@ -39,7 +39,6 @@
     remote_src: yes
 
 - include_tasks: download_cobbler_requires.yml
```

Comment:
Oh, I see what you did there. So why do we need the CentOS DVD on the deployer node for non-cobbler deployments ?

Reply:
The CentOS DVD is mounted on the deployer and served as a repo to the CentOS computes as in [1].

1. https://github.com/SUSE-Cloud/automation/pull/2956/files#diff-882ace793baa149c2b8cad9c43889d6bR160